conf/routes

```diff
@@ -14,4 +14,6 @@ GET     /api/search/basic      @controllers.HostInfoController.simpleStringSearc
 POST    /api/hostinfo                   @controllers.HostInfoController.addRecord
 
 POST    /api/logins             @controllers.LoginHistoryController.updateLogins
-GET     /api/logins/:hostname   @controllers.LoginHistoryController.loginsFor(hostname,limit:Option[Int])
\ No newline at end of file
+GET     /api/logins/:hostname   @controllers.LoginHistoryController.loginsFor(hostname,limit:Option[Int])
++ nocsrf
```

Comment:
agh, this was it.  let's leave it in for the time being (not like it's an internet-facing service anyway....) but should sort that in the future